### PR TITLE
Plain English

### PR DESCRIPTION
**Pull request by brianteeman**
The accessibility standard Web Content Accessibility Guidelines (WCAG) 2.0 Section 3 states https://www.w3.org/TR/WCAG20/#understandable

>### 3. Understandable
> Information and the operation of user interface must be understandable.

>#### Guideline 3.1 Readable
> Make text content readable and understandable.

> #### Success Criterion 3.1.5 Reading Level
> When text requires reading ability more advanced than the lower secondary education level after removal of proper names and titles, supplemental content, or a version that does not require reading ability more advanced than the lower secondary education level, is available.

To achieve this we should use "plain language" wherever possible

1. Word choice: use the simplest word that conveys your meaning.
http://plainlanguagenetwork.org/plain-language/what-is-plain-language/

2. Prefer the short word to the long.
https://www.plainlanguage.gov/guidelines/words/

This PR is an initial step of improving the strings to match this accessibility guideline. It is continues the work of some previous PR eg  #18852 remove "just" and #14607 remove "successfully"

Sources referred to in creating this PR include RESOURCES
https://www.plainlanguage.gov/guidelines/words/use-simple-words-phrases/

### CODE REVIEW

File: language/en-GB/en-GB.mod_languages.ini [hunk 1]
```diff
@@ -28,4 +28,4 @@ MOD_LANGUAGES_OPTION_DEFAULT_LANGUAGE="Default"
...
...e parameter is set for the Language filter plugin, it will redirect to the associated item in the language chosen. Thereafter, the navigation will be the one defined for that language. <br />If the plugin <strong>'System - Language Filter'</strong> is disabled, this may have unwanted results.<br /><strong>Method:</strong><br />1. Open Language Manager Content tab and make sure the Languages you want to use in contents are published and have a Language Code for the URL as well as prefix for the image used in the module display.<br />2. Create a Home page by assigning a language to a menu item and defining it as Default Home page for each published content language. <br />3. Thereafter, you can assign a language to any Article, Category, Module, News Feed, Web Links in Joomla.<br />4. Make sure the module is published and the plugin is enabled. <br />5. When using associated items, make sure the module is displayed on the pages relevant. <br />6. The way the flags or names of the languages are displayed is defined by the ordering in the Language Manager - Content Languages.<br ><br >If this module is published, it is suggested to publish the Administrator multilingual status module."
```
Comment (Quy) [inline language/en-GB/en-GB.mod_languages.ini]: Change to `relevant pages`.

File: language/en-GB/en-GB.mod_languages.sys.ini [hunk 2]
```diff
@@ -4,6 +4,6 @@
...
...e parameter is set for the Language filter plugin, it will redirect to the associated item in the language chosen. Thereafter, the navigation will be the one defined for that language. <br />If the plugin <strong>'System - Language Filter'</strong> is disabled, this may have unwanted results.<br /><strong>Method:</strong><br />1. Open Language Manager Content tab and make sure the Languages you want to use in contents are published and have a Language Code for the URL as well as prefix for the image used in the module display.<br />2. Create a Home page by assigning a language to a menu item and defining it as Default Home page for each published content language. <br />3. Thereafter, you can assign a language to any Article, Category, Module, News Feed, Web Links in Joomla.<br />4. Make sure the module is published and the plugin is enabled. <br />5. When using associated items, make sure the module is displayed on the pages relevant. <br />6. The way the flags or names of the languages are displayed is defined by the ordering in the Language Manager - Content Languages.<br ><br >If this module is published, it is suggested to publish the Administrator multilingual status module."
```
Comment (Quy) [inline language/en-GB/en-GB.mod_languages.sys.ini]: Change to `relevant pages`.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ed49ad07ad5440c2ac6d9f35085e753ee9c1cb78<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19629">issues.joomla.org/tracker/joomla-cms/19629</a>.</sub>

**Comment by anuragteapot** [comment]
I have tested this item :white_check_mark: successfully on ed49ad07ad5440c2ac6d9f35085e753ee9c1cb78<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19629">issues.joomla.org/tracker/joomla-cms/19629</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Comment by zero-24** [comment]
Merged. Thanks 👍

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**